options/locale/locale_en-US.ini

```diff
@@ -746,12 +747,12 @@ passcode_invalid = The passcode is incorrect. Try again.
 twofa_enrolled = Your account has been enrolled into two-factor authentication. Store your scratch token (%s) in a safe place as it is only shown once!
 twofa_failed_get_secret = Failed to get secret.
 
-u2f_desc = Security keys are hardware devices containing cryptographic keys. They can be used for two-factor authentication. Security keys must support the <a rel="noreferrer" href="https://fidoalliance.org/">FIDO U2F</a> standard.
-u2f_register_key = Add Security Key
-u2f_nickname = Nickname
-u2f_press_button = Press the button on your security key to register it.
-u2f_delete_key = Remove Security Key
-u2f_delete_key_desc = If you remove a security key you can no longer sign in with it. Continue?
+webauthn_desc = Security keys are hardware devices containing cryptographic keys. They can be used for two-factor authentication. Security keys must support the <a rel="noreferrer" href="https://w3c.github.io/webauthn/#webauthn-authenticator">WebAuthn Authenticator</a> standard.
```

Comment:
It's better to have `_blank="blank"` to open a new window.

Reply:
I wonder if we can somehow remove such attributes from translations and handle it in a more generic way during rendering.

Generally speaking, I'd say all external links should have `target="_blank"` and `rel="noreferrer noopener"`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

```
Setting target="_blank" on <a> elements now implicitly provides the same rel behavior as setting rel="noopener" which does not set window.opener. 
```

So `target="_blank"` should be enough